modules/common/route/types.go

```diff
@@ -22,16 +22,20 @@ import (
 	"time"
 
 	routev1 "github.com/openshift/api/route/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // Route -
+// +kubebuilder:object:generate:=false
```

Comment:
why don't we want these in zz_generated?

Reply:
it is not a struct which we use in the API